containers/unit_tests/CMakeLists.txt

```diff
@@ -13,23 +13,29 @@ foreach(Tag Threads;Serial;OpenMP;HPX;Cuda)
   string(TOLOWER ${Tag} dir)
   # Add test for that backend if it is enabled
   if(Kokkos_ENABLE_${DEVICE})
-    KOKKOS_ADD_EXECUTABLE_AND_TEST(
-      UnitTest_${Tag}
-      SOURCES
-        UnitTestMain.cpp
-        ${dir}/Test${Tag}_BitSet.cpp
-        ${dir}/Test${Tag}_DualView.cpp
-        ${dir}/Test${Tag}_DynamicView.cpp
-        ${dir}/Test${Tag}_DynRankViewAPI_generic.cpp
-        ${dir}/Test${Tag}_DynRankViewAPI_rank12345.cpp
-        ${dir}/Test${Tag}_DynRankViewAPI_rank67.cpp
-        ${dir}/Test${Tag}_ErrorReporter.cpp
-        ${dir}/Test${Tag}_OffsetView.cpp
-        ${dir}/Test${Tag}_ScatterView.cpp
-        ${dir}/Test${Tag}_StaticCrsGraph.cpp
-        ${dir}/Test${Tag}_UnorderedMap.cpp
-        ${dir}/Test${Tag}_Vector.cpp
-        ${dir}/Test${Tag}_ViewCtorPropEmbeddedDim.cpp
-      )
+    set(UnitTestSources UnitTestMain.cpp)
+    set(dir ${CMAKE_CURRENT_BINARY_DIR}/${dir})
+    file(MAKE_DIRECTORY ${dir})
```

Comment:
Was there a reason for making a directory rather prepending dir_ to the filename? Doesn't need to change. Just curious.

Reply:
No reason